pyproject.toml

```
@@ -17,31 +17,37 @@ dynamic = ["version"]
 license = "Apache-2.0"
 name = "xarray"
 readme = "README.md"
-requires-python = ">=3.10"
+requires-python = ">=3.11"
 
-dependencies = ["numpy>=1.24", "packaging>=23.2", "pandas>=2.1"]
+dependencies = ["numpy>=1.26", "packaging>=24.0", "pandas>=2.2"]
 
 # We don't encode minimum requirements here (though if we can write a script to
 # generate the text from `min_deps_check.py`, that's welcome...). We do add
 # `numba>=0.54` here because of https://github.com/astral-sh/uv/issues/7881;
 # note that it's not a direct dependency of xarray.
 
 [project.optional-dependencies]
-accel = ["scipy", "bottleneck", "numbagg", "numba>=0.54", "flox", "opt_einsum"]
+accel = [
+  "scipy>=1.13",
+  "bottleneck",
+  "numbagg>=0.8",
+  "numba>=0.59",
+  "flox>=0.9",
+  "opt_einsum",
+]
 complete = ["xarray[accel,etc,io,parallel,viz]"]
 io = [
-  "netCDF4",
+  "netCDF4>=1.6.0",
   "h5netcdf",
-  "scipy",
-  'pydap; python_version<"3.10"',
-  "zarr",
+  "scipy>=1.13",
```

```suggestion
  "pydap",
  "scipy>=1.13",
```

pydap now supports 3.10+